ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditTextInputConnectionWrapper.java

```diff
@@ -57,6 +57,7 @@
   private EventDispatcher mEventDispatcher;
   private boolean mIsBatchEdit;
   private @Nullable String mKey = null;
+  private int previousComposingTextLength;
```

Comment:
This commit fixes an edge case issue introduced by the [previous commit](https://github.com/facebook/react-native/pull/30751/commits/dfe940205bbb7a10c93a7a34afdd3e4a53855fcd). 

When `maxLength` is set to 1 or more and the max length is reached, the composing text never changes. So backspace can be inferred correctly. However, when `maxLength` is set to 0 then whatever the user types is appended to the composing text and therefore we can no longer infer the backspace keypress.

The videos below demonstrate the issue and the result of the fix introduced in this commit.
In both theses videos, `maxLength` is set to 0.

### Before


https://user-images.githubusercontent.com/24393922/104837671-105fab80-58f1-11eb-9a7f-c293299e5a54.mov


### After


https://user-images.githubusercontent.com/24393922/104837919-c4ae0180-58f2-11eb-81dc-fd199357238b.mov